### PR TITLE
docs: no network request for performance strategy

### PR DESCRIPTION
**Pull request by Jun711**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
It adds clarification to the users that no network requests will be sent if strategy is performance. Some might be confused that cache is first used while waiting for a network request.

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by brandonroberts** [comment]
Thanks for the PR @Jun711! You must [Sign the CLA](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#cla) before this PR can move forward.

**Comment by Jun711** [comment]
I signed it!

File: aio/content/guide/service-worker-config.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ For example, the string `5s30u` will translate to five seconds and 30 millisecon
 
 The Angular service worker can use either of two caching strategies for data resources.
 
-* `performance`, the default, optimizes for responses that are as fast as possible. If a resource exists in the cache, the cached version is used. This allows for some staleness, depending on the `maxAge`, in exchange for better performance. This is suitable for resources that don't change often; for example, user avatar images.
+* `performance`, the default, optimizes for responses that are as fast as possible. If a resource exists in the cache, the cached version is used, there will be no request from the network at all. This allows for some staleness, depending on the `maxAge`, in exchange for better performance. This is suitable for resources that don't change often; for example, user avatar images.
```
Comment (brandonroberts) [inline aio/content/guide/service-worker-config.md]:
```suggestion
* `performance`, the default, optimizes for responses that are as fast as possible. If a resource exists in the cache, the cached version is used, and no network request is made. This allows for some staleness, depending on the `maxAge`, in exchange for better performance. This is suitable for resources that don't change often; for example, user avatar images.
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]

**Review by brandonroberts — APPROVED** [review]

**Review by gkalpak — APPROVED** [review]
The commit message should be `docs(service-worker): ...`.
Other than that LGTM 👍

**Pull request closed without merging**